tardis/montecarlo/montecarlo_numba/tests/test_cuda_formal_integral.py

```diff
@@ -21,7 +21,7 @@
     NumbaFormalIntegrator,
 )
 
-from tardis.montecarlo.base import MontecarloTransport
+from tardis.montecarlo.base import MontecarloTransportSolver
```

Comment:
Is this used here? I don't see the class name updated in the file

Reply:
ran ruff. This is skipped on non-cuda machines and we should track that.